raiden/network/proxies/secret_registry.py

```diff
@@ -71,6 +75,22 @@ def __init__(self, jsonrpc_client, secret_registry_address, contract_manager: Co
     def register_secret(self, secret: Secret):
         self.register_secret_batch([secret])
 
+    def register_secret_light(self, signed_tx: SignedTransaction):
+        try:
+            log.debug("register_secret_light called")
+            transaction_hash = self.proxy.broadcast_signed_transaction(signed_tx)
+            self.client.poll(transaction_hash)
```

Comment:
I think we already have something to wait for a transaction and handle the result but i'm not sure

Reply:
I don't know if we had, but I added something to prevent us from copying and pasting this over and over again. 
I added it later than this, so I forgot to merged it with this. I'll change this